### PR TITLE
Post event changes to home page

### PR DESCRIPTION
**Pull request by danielfspencer**
image

image

### CODE REVIEW

File: _includes/hero.html [hunk 1]
```diff
@@ -20,33 +20,26 @@
...
+                        <button type="button" class="btn btn-primary"><i class="fab fa-discord"></i> Join Discord</button>
                     </a>
-                    <a href="{{ page.socials.devpost }}">
-                        <button type="button" class="btn btn-primary"><i class="fas fa-code"></i> 2. Register on Devpost</button>
+                    <a href="{{ page.socials.donate }}">
+                        <button type="button" class="btn btn-primary"><i class="fas fa-money-check-alt"></i> Donate to WHO Fund</button>
                     </a>
                     <a href="{{ page.socials.schedule }}">
-                        <button type="button" class="btn btn-primary"><i class="fas fa-calendar"></i> 3. View Schedule</button>
+                        <button type="button" class="btn btn-primary"><i class="fas fa-calendar"></i> View Schedule</button>
```
Comment (JFKingsley) [inline _includes/hero.html]: does the schedule even matter at this point?

**Review by JFKingsley — DISMISSED** [review]

File: _includes/hero.html (hunk 1 above)
Comment (danielfspencer) [reply _includes/hero.html]: might be useful for seeing what workshops there were?

Comment (danielfspencer) [reply _includes/hero.html]: we can change this later if needed, i'm going to get this merged for now

**Review by wrussell1999 — APPROVED** [review]

File: _includes/hero.html (hunk 1 above)
Comment (wrussell1999) [reply _includes/hero.html]: Removed

**Pull request merged**